build-aux/com.belmoussaoui.GtkRustTemplate.Devel.json

```diff
@@ -1,11 +1,11 @@
 {
     "id": "com.belmoussaoui.GtkRustTemplate.Devel",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "47",
+    "runtime-version": "49",
     "sdk": "org.gnome.Sdk",
     "sdk-extensions": [
         "org.freedesktop.Sdk.Extension.rust-stable",
-        "org.freedesktop.Sdk.Extension.llvm18"
+        "org.freedesktop.Sdk.Extension.llvm20"
```

Comment:
I know that Rust `1.92` uses `llvm20`, but I wonder would `llvm21` work for that?